test/test.sh

```diff
@@ -0,0 +1,8 @@
+#/bin/bash
```

Comment:
1. Nit: There is not bash-specific here (any bourne shell compatible interpreter may run this code), so I would use `#!/bin/sh`. And, yep, the exclamation mark was forgotten.
2. I usually name such files with `.example` postfix and add a name without `.example` to gitignore. So a user may keep the repository clean and have secrets in a gitignored file. See token.txt.example [here](https://github.com/Totktonada/closed_issues) for example.
3. I would not name it 'test', because it does not verify correctness of the implementation. (The same for attach.test.lua.) I'll propose a name if something good will come into my mind.

Reply:
The file was replaced by the actual test in PR #23, so the comment is not actual anymore.